src/store/index.js

```diff
@@ -1,17 +1,26 @@
 import { createStore, applyMiddleware } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import thunkMiddleware from 'redux-thunk';
-import rootReducer from './reducers/index';
+import promiseMiddleware from 'redux-promise-middleware';
+import rootReducer from '../reducers/index';
 
 const composeEnhancers = composeWithDevTools({
   // Specify name here, actionsCreators and other options if needed here
 });
 
 const storeConfig = () => {
+  const middlewares = [
+    thunkMiddleware,
+    promiseMiddleware(),
```

Comment:
Are you using this middleware anywhere?

Reply:
Yes, because in reducers I have 3 cases of Pending, Fulfilled and Rejected. So when the action is dispatched, it checks for the 3 statuses.